### PR TITLE
Add logging for worker bundle is detected

### PR DESCRIPTION
**Pull request by maiqbal11**
Adding logging so we can track how worker bundle usage changes over time.

### CODE REVIEW

**Comment by maiqbal11** [comment]
\cc @anirudhgarg 

Circling back on the chat that @ankitkumarr and I had discussed that the current change might not account for users who do not experience cold start. We should be fine to put this logging in the path where it currently lives. Seabreeze/App Service VM updates should (with high probability) trigger cold starts within the 1 month window that we are giving users to move away from the bundle. Other kinds of host changes risk being hacky and/or too involved for the scope of something that will be removed soon.

File: src/WebJobs.Script.WebHost/Management/InstanceManager.cs [hunk 1]
```diff
@@ -169,6 +169,12 @@ private async Task ApplyContext(HostAssignmentContext assignmentContext)
                     ZipFile.ExtractToDirectory(filePath, options.ScriptPath, overwriteFiles: true);
                     _logger.LogInformation($"Zip extraction complete");
                 }
+
+                string bundlePath = Path.Combine(options.ScriptPath, "worker-bundle");
+                if (Directory.Exists(bundlePath))
+                {
+                    _logger.LogInformation($"Worker bundle detected");
```
Comment (soninaren) [inline src/WebJobs.Script.WebHost/Management/InstanceManager.cs]: Add the string to resourcefiles

**Comment by maiqbal11** [comment]
Work item to remove this change: https://github.com/Azure/azure-functions-host/issues/4282

**Comment by maiqbal11** [comment]
@soninaren, merged in. You should be able to pick this up for the release

**Pull request merged**